src/async_impl/response.rs

```diff
@@ -75,6 +75,20 @@ impl Response {
         self.res.headers_mut()
     }
 
+    /// Get the trailers of this `Response`.
+    pub fn trailers(&mut self) -> Option<&HeaderMap> {
```

Comment:
```suggestion
    pub fn trailers(&self) -> Option<&HeaderMap> {
```

Sorry if I'm missing something but it looks like this doesn't need a mutable reference